docs/flask.md

```diff
@@ -0,0 +1,118 @@
+# Flask-GraphQL
+
+Adds GraphQL support to your Flask application.
+
+## Installation
+
+To install the integration with Flask, run the below command on your terminal.
+
+`pip install graphql-server-core[flask]`
+
+## Usage
+
+Use the `GraphQLView` view from `graphql_server.flask`.
+
+```python
+from flask import Flask
+from graphql_server.flask import GraphQLView
+
+from schema import schema
+
+app = Flask(__name__)
+
+app.add_url_rule('/graphql', view_func=GraphQLView.as_view(
+    'graphql',
+    schema=schema,
+    graphiql=True,
+))
+
+# Optional, for adding batch query support (used in Apollo-Client)
+app.add_url_rule('/graphql/batch', view_func=GraphQLView.as_view(
+    'graphql',
+    schema=schema,
+    batch=True
+))
+
+if __name__ == '__main__':
+    app.run()
+```
+
+This will add `/graphql` and `/graphiql` endpoints to your app.
+
+### Special Note for GraphQL-Server V3
+
+If you are using the `Schema` type of [Graphene](https://github.com/graphql-python/graphene) library, be sure to use the `graphql_schema` attribute to pass as schema on the `GraphQLView` view. Otherwise, the `GraphQLSchema` from `graphql-core` is the way to go.
+
+**Graphene Schema**
+
+```python
+from graphene import ObjectType, String, Schema
+
+
```

Comment:
Nit pick: in code examples I don’t think it’s necessary to have 2 lines before a class definition.